Flow.Launcher/Notification.cs

```diff
@@ -8,10 +9,17 @@ namespace Flow.Launcher
 {
     internal static class Notification
     {
+        internal static bool legacy = Environment.OSVersion.Version.Build < 19041;
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Interoperability", "CA1416:Validate platform compatibility", Justification = "<Pending>")]
+        internal static void Uninstall()
+        {
+            if (!legacy)
+                ToastNotificationManagerCompat.Uninstall();
```

Comment:
What does this Uninstall do?

Reply:
https://docs.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/send-local-toast?tabs=uwp#step-4-handling-uninstallation